File: frontend/project_frontend/src/app/course.ts
export class Course {
courseId:number=0;
teacherId:number=0;
courseName: string='';
instrument: string='';
courseDesc: string='';
videoUrl: string='';
constructor(courseId:number,teacherId:number,courseName:string,instrument:string,courseDesc:string,videoUrl:string){
    this.courseId=courseId;
    this.teacherId=teacherId;
    this.instrument=instrument;
    this.courseName=courseName;
    this.courseDesc=courseDesc;
    this.videoUrl=videoUrl;
}

}

File: maestromatch/content-management-service/src/main/java/com/stackroute/contentmanagementservice/model/Course.java
package com.stackroute.contentmanagementservice.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.annotation.Transient;
import org.springframework.data.mongodb.core.mapping.Document;



@Document (collection = "Courses")
public class Course {

	@Transient
	public static final String SEQUENCE_NAME="course_sequence";
	
	@Id	
	private int courseId;
	private int teacherId;

	public int getTeacherId() {
		return teacherId;
	}

	public void setTeacherId(int teacherId) {
		this.teacherId = teacherId;
	}

	private String courseName;
	private String instrument;

	public String getInstrument() {
		return instrument;
	}

	public void setInstrument(String instrument) {
		this.instrument = instrument;
	}

	private String courseDesc;
	private String videoUrl;
	
	public String getVideoUrl() {
		return videoUrl;
	}
	public void setVideoUrl(String videoUrl) {
		this.videoUrl = videoUrl;
	}
	public int getCourseId() {
		return courseId;
	}
	public void setCourseId(int courseId) {
		this.courseId = courseId;
	}
	public String getCourseName() {
		return courseName;
	}
	public void setCourseName(String courseName) {
		this.courseName = courseName;
	}
	public String getCourseDesc() {
		return courseDesc;
	}
	public void setCourseDesc(String courseDesc) {
		this.courseDesc = courseDesc;
	}
	
	
	
}
